Src/Application/CaseStudy.API/Controllers/UserFavController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using CaseStudy.Core.Contracts.IUnitOfWork;
+using CaseStudy.Core.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+
+namespace CaseStudy.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserFavController : ControllerBase
+    {
+        private IUserFavServices userFavServices;
+        public UserFavController(IUserFavServices favServices)
+        {
+            userFavServices = favServices;
+        }
+
+        [HttpGet]
```

Comment:
I think your url is **"UserFav?userId"** but instead of ?userId query parameter, can we go with route parameter? https://learn.microsoft.com/en-us/azure/architecture/best-practices/api-design

Reply:
okay